recipes/faac/all/conandata.yml

```
@@ -8,4 +8,5 @@ sources:
 patches:
   "1.30":
     - patch_file: "patches/001-fix-out-of-root-build.patch"
-      base_path: "source_subfolder"
+      patch_description: "Fix out of root build"
```

https://github.com/conan-io/conan-center-index/blob/39b8177ad0ecbec6aa91fbaf174c1ecec8de0859/recipes/faac/all/patches/001-fix-out-of-root-build.patch#LL6C6-L6C101

```suggestion
      patch_description: "Fix out of root build"
      patch_source: "https://github.com/m-ab-s/media-autobuild_suite/commit/11d58abd8fde7c70378993029bad7bbaeb5f7e3e"
```

---

And when you look at this link, it seems totally unrelated. This is why I've skipped this field.

---

🙈 Quality Control ... why did that go in the first place lol -- Okay well I answered my own question :)

---

this patch maybe https://github.com/knik0/faac/commit/c8d12a5c7c5b6f1c4593f0a6c1eeceacc4d7c941.patch?

---

😈 

```suggestion
      patch_description: "Fix out of root build"
      patch_source: "https://github.com/knik0/faac/commit/c8d12a5c7c5b6f1c4593f0a6c1eeceacc4d7c941.patch"
```

Now we can add it 🚀